src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -78,6 +97,10 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
     setIsLicenseModalOpen(!isLicenseModalOpen);
   };
 
+  const toggleEmbargoModal = () => {
+    setIsEmbargoModalOpen(!isEmbargoModalOpen);
+  };
```

Comment:
skikkelig pirkepirk, men for å fjerne to linjer:
```suggestion
  const toggleEmbargoModal = () => setIsEmbargoModalOpen(!isEmbargoModalOpen);
```

kan gjøre samme med funksjonen over også